test/Tizen.NUI.Samples/Tizen.NUI.Samples/Samples/VideoViewTest.cs

```diff
@@ -0,0 +1,135 @@
+﻿
+using global::System;
+using Tizen.NUI.BaseComponents;
+
+namespace Tizen.NUI.Samples
+{
+    using tlog = Tizen.Log;
+
+    public class VideoViewTest : IExample
+    {
+        public class myPlayer : Tizen.Multimedia.Player
+        {
+            public myPlayer() : base()
+            {
+                //Initialize();
+            }
+
+            public myPlayer(IntPtr p) : base(p, null)
+            {
+                //Initialize();
+            }
+        }
+
+        Window win;
+        myPlayer player;
+        string resourcePath;
+        const string tag = "NUITEST";
+        public void Activate()
+        {
+            win = Window.Instance;
+            win.BackgroundColor = Color.Green;
+            win.KeyEvent += Win_KeyEvent;
+            win.TouchEvent += Win_TouchEvent;
+
+            var dummy = new View();
+            dummy.Size2D = new Size2D(200, 200);
```

Comment:
please use Size(200,200)

Reply:
Hello, I changed as Size(200, 200). thank you.